### DISCUSSION TITLE
RDS snapshot upload

### DISCUSSION THREAD

**Discussion by Backscratcher** [aws/aws-cli#7425] [Q&A] [answered]
Hi!

Is there a way of creating your own database snapshots? Consider this situation, imagine you would like to use last snapshot from production in some other environment but first you would like to do some operations on it like clean sensitive data, anonymize some columns etc. Afterwards you would update old/create new snapshot. Is there a way of doing this using aws-cli or does this require some custom work with involving s3.

**Comment by RyanFitzSimmonsAK** [comment, marked as answer]
Hi @Backscratcher, thanks for reaching out.  Creating a snapshot of a database is absolutely possible, here's a [guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateSnapshot.html) and the associated [documentation](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/rds/create-db-snapshot.html).  Please let me know if that isn't what you're looking for, or if you have any other questions.
